Move LHCb to use Draco and LHCb.glb

EdwardMoyse opened the pull request:
This migrates LHCb to compressed glb (generated with: `gltf-pipeline -i LHCb.gltf -o LHCb.glb -d`)

This reduces the size of the geometry from 19Mb to 7.2Mb

To uncompress the geometry, we need to add the `DRACOLoader` (see the discussion in #529)

In my local tests this doesn't really improve anything. However I would expect it might running remotely... however I need to test this (hopefully I can use the deployment here to do so).

cc @sponce

EdwardMoyse commented:
Okay, comparing surge ( http://phoenix-pr-530.surge.sh/#/lhcb) vs normal (http://hepsoftwarefoundation.org/phoenix/#/lhcb) I see:
```
Surge:  4.5s, 4.3s, 3.8s
Normal: 3.6s, 4.6s, 4.3s.
```

So, I don't think it makes it any faster (at least on my very quick connection), but then I still think this might be worth it, because this means the overhead of decompressing isn't too bad... but for people on slower connections it might help a lot. The only counter-point is this is a M1 Pro mac, so pretty fast.

<img width="593" alt="image" src="https://user-images.githubusercontent.com/6764617/208635633-cd0b3275-641b-481b-95df-a9a9fa960f36.png">

9inpachi commented:
You can check [how to throttle network using Chrome DevTools](https://www.browserstack.com/guide/how-to-perform-network-throttling-in-chrome) and see how long it takes for the geometry to load on a slow network. I tried on the surge URL for PR but the application is throwing errors.

sponce commented:
I confirm I see surge slightly slower on the CERN fast connection. No big news here and I agree we could merge. One thing though, don't you want to drop the old file ? Otherwise the gltf and the glb will quickly become out of sync. And the other missing bit is maybe the recipe for creating the glb form the gltf, for future updates

bencouturier commented:
I agree it's best to keep only one and that we should document the conversion! Looks very useful for slow connections, many thanks.

EdwardMoyse commented:
Hi @sponce and @bencouturier - I removed the `gltf`file now (I didn't want to do it without your permission) and added some new geometry documentation (feel free to add to it!)

sponce commented on packages/phoenix-event-display/src/managers/three-manager/import-manager.ts:
```diff
@@ -180,6 +182,13 @@ export class ImportManager {
+
+    const dracoLoader = new DRACOLoader();
+    dracoLoader.setDecoderPath(
+      `https://cdn.jsdelivr.net/npm/three@0.${REVISION}.0/examples/js/libs/draco/`
```
I've just checked and your path is correct for 146... It seems things have changed on the threejs side !

sponce replied:
And here is the explanation : https://discourse.threejs.org/t/the-examples-js-directory-will-be-removed-with-r148/45349

EdwardMoyse replied:
Right - updating `threejs` version is always a bit of work. Since phoenix is on [0.147.1](https://github.com/HSF/phoenix/blob/main/package.json#L28) I would definitely not recommend moving your local deployment to something ahead of that. If you need it for some reason, we can prioritise updating phoenix.

sponce replied:
My issue is that we did not intend to move our local deployment. I have to check why we do not have the default 147. @bencouturier any idea ?
